### PR TITLE
clean up docs some more

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@rhcarvalho @mfojtik ptal.

File: README.md [hunk 1]
```diff
@@ -1,36 +1,30 @@
 Rails Sample App on OpenShift
 ============================
 
-This is a quickstart Rails application for OpenShift v3.
+This is a quickstart Rails application for OpenShift v3 that you can use as the starting point to develop your own and deploy it on an [OpenShift](https://github.com/openshift/origin) cluster.
```
Comment (jhadvig) [inline README.md]: Suggestion - `... that you can use as a starting point to develop your own application and deploy it in OpenShift cluster`
... not a native speaker, just sounds better in my head :P

File: public/index.html [hunk 2]
```diff
@@ -274,6 +277,13 @@
                   the *_SERVICE_HOST and *_SERVICE_PORT environment variables will be automatically
                   created.
                   </p>
+                  <p>
+                  You will need to redeploy your application in order to pick up the new environment variables, you can force a deployment
```
Comment (jhadvig) [inline public/index.html]: Maybe split the sentence after the comma ?

**Comment by jhadvig** [comment]
Just two small nits that might be just matter of taste. Otherwise LGTM

**Comment by bparees** [comment]
nits addressed.

**Pull request merged**